systemtest/src/test/java/io/strimzi/systemtest/rollingupdate/AlternativeReconcileTriggersST.java

```diff
@@ -50,47 +50,51 @@
     @Test
     void testManualTriggeringRollingUpdate() {
         String topicName = KafkaTopicUtils.generateRandomNameOfTopic();
+        String clusterName = CLUSTER_NAME + "-manual-rolling";
```

Comment:
It's only a question, but why do we need to rename it? Was there some issue?

Reply:
Yes, I think the second test get same PV (or something like that) and CO wasn't able to roll some pods.

Reply:
That make sense :) thanks for explanation.